docker/docker-compose.suite-desktop-test.yml

```diff
@@ -20,3 +20,9 @@ services:
     volumes:
       - ../:/trezor-suite
       - /tmp/.X11-unix:/tmp/.X11-unix:rw
+
+  electrum-regtest:
+    image: ghcr.io/vdovhanych/electrs:latest
```

Comment:
Use this image instead, please - `ghcr.io/trezor/electrs:latest`

Reply:
https://github.com/trezor/trezor-suite/pull/6258/commits/d3e232a4ebd5775b01788dc0cde5d7e71475e5dc